_RFCs/0003-tracking-package-versions.md

```diff
@@ -0,0 +1,56 @@
+- Change Name: standardize_tracking_major_package_versions
+- Start Date: 2018-03-27
+- RFC PR: (leave this empty)
+- Core-Plans Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+Everytime a new major version of software packaged by Habitat is released (whether stable or an edge release), a new core plan must be made for that version (i.e. when Docker11 is released, we should create core/docker11).
+
+Generic package names (ones which do not specify a version in the name - i.e. core/docker) must always point to the latest **stable** version of the software.  They must not track edge versions of the software.  They must source the plan for the latest stable plan which tracks that software (i.e. if the current stable version of Docker is Docker10, and the edge version is Docker11, core/docker should source core/docker10).
+
+Edge versions of software are not tracked by core plans. If a user wishes to use the edge version of a package, they must either fork an existing core plan or create a new core plan to package it. Core plans only support stable versions of software.
+
+# Motivation
+[motivation]: #motivation
+
+We currently have several generic packages (i.e. core/docker) that track the current stable version of the packaged software (as of today, core/docker points to Docker 17). When a new major version of the package becomes available, we move the old version (i.e. Docker 16) to a separate plan to track the old version (i.e. core/docker16) while the generic package (core/docker) will be updated to point to the current major version of the software.
+
+However, we do not have a process in place to handle edge releases of software. As of today, Docker 18 is the latest edge version of Docker, but the latest stable version is still Docker 17. This brought us to the question - should "core/docker" track the latest stable version or the latest edge version? Should we even track edge versions at all?
```

Comment:
This paragraph feels irrelevant after the paragraph right above "motivation" that states explicitly that edge versions of software are not tracked by core plans.